src/data.json

```diff
@@ -1,142 +1,62 @@
 [
   {
-    "image_url": "http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg",
-    "title": "UniWhal",
-    "description": "A unicorn and a narwhal nuzzling their horns",
-    "keyword": "narwhal",
-    "horns": 1
+    "image_url": "http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg"
   },
```

Comment:
Ok, so this isn't great. You don't want to tamper with the JSON data you've received. Your best bet here would be to go back to Lab 1, and copy paste all the JSON data back into this file.
One of the things you're supposed to do for this lab is display information about each beast. So not just an image, you'll also want to have the name/title of beast displayed. Later on, in future labs, you'll also need to be able to filter the beasts by how many horns they have, so if you've removed the property "horns" from the JSON file you'll have no way to filter through the beasts in a later lab.